public/video-ui/src/components/ReactApp.js

```diff
@@ -59,7 +59,12 @@ class ReactApp extends React.Component {
           currentPath={this.props.location.pathname}
           video={this.props.video || {}}
           publishedVideo={this.props.publishedVideo || {}}
-          showPublishedState={this.props.params.id ? true : false}
+          showPublishedState={
+            this.props.params.id ||
+              this.props.location.pathname === '/videos/create'
```

Comment:
Otherwise we'd get a `Create new video` button in the header when creating a video which doesn't make much sense. Since it is the same video data form that we are displaying, I think it makes sense to display the same header as when editing the atom.